tests/unit_test/preprocess/test_arci_prerpocessor.py

```diff
@@ -20,7 +20,8 @@ def test_arci_preprocessor_noembed(train_inputs, validation_inputs):
     assert len(rv_train.left) == 1
```

Comment:
Why not directly assert the `value` rather than the `len`? e.g., `asset rv_train.left == [...]`.